dev-tools/markdown.style.rb

```diff
@@ -0,0 +1,5 @@
+# Include all rules as a starting point
+all
+
+# Override rules' config
+rule 'MD026', :punctuation => '.,;:'
```

Comment:
The difference from default config is that `!` and `?` are allowed (see first header of `CONTRIBUTING.md`).